### ISSUE TITLE
joining channels i'm not in

### ISSUE THREAD

**Issue by bigmittens** [sm00th/bitlbee-discord#49]
whenever discord logs in, i'm added to channels on servers on discord that my role doesn't have access too. i don't receive messages from these channels, but buffers still open for each of them.

**Comment by sm00th** [comment]
Yes, there is no support for permissions whatsoever in bitlbee-discord. Issue #34 discusses a possible solution for the problem with joining unwanted channels.
Implementing permission support is something I've been thinking about for a while. Not joining/hiding the channels you have no access to is the only use bitlbee-discord can make of permissions and I am not sure if implementing those is worth the time it will take.
